starter/script.js

```diff
@@ -1 +1,301 @@
+  went out and when they came back on you seemed to be...dead!
+  How inconvenient that must be for you.  Anyhow, lets see if
+  you can remember who it was that committed this "heinous" crime.`)
+  return firstClue ()
+}
+
+function firstClue () {
+  var room = prompt(`Do you happen to remember which room you were killed in?
+
+      Was it the
+      [Conservatory],
+      [Dining Room],
+      [Hall],
+      or [Billiard Room]?`)
+
+  switch (room) {
+    case 'Conservatory':
+      return conservatory()
+      break
+    case 'Dining Room':
+      return diningRoom()
+      break
+    case 'Hall':
+      return hall()
+      break
+    case 'Billiard Room':
+      return billiardRoom()
+      break
+    default:
+      prompt("I don't think thats a room in this house...")
+      return firstClue()
+    }
+}
+
+function conservatory () {
+  var sex = prompt(`At that time, in the Conservatory, Mrs. White, Professor Plum,
+      Mrs. Peacock, Colonel Mustard, and you were searching for a flashlight and
+      then you were killed.
+
+      Do you remember if the killer was a [Man] or a [Woman]`)
+    if (sex === 'Man') {
+      return conservatoryMan()
+    } else if (sex === 'Woman') {
+      return conservatoryWoman()
+    } else {
+      alert('Sorry try again!')
+      conservatory()
+    }
+}
+
+function diningRoom () {
+  var sex = prompt(`At that time, in the Dining Room, Mrs. Peacock, Mr. Green,
+      Ms. Scarlet, Colonel Mustard, and you were searching for candles and
+      then you were killed.
+
+      Do you remember if the killer was a [Man] or a [Woman]`)
+    if (sex === 'Man') {
+      return diningRoomMan()
+    } else if (sex === 'Woman') {
+      return diningRoomWoman()
+    } else {
+      alert('Sorry try again!')
+      diningRoom()
+    }
+}
+
+function hall () {
+  var sex = prompt(`At that time, in the Hall, Mrs. Peacock, Colonel Mustard,
+      Mrs. White, Professor Plumb, and you were searching for the fuse box and
+      then you were killed.
+
+      Do you remember if the killer was a [Man] or a [Woman]`)
+    if (sex === 'Man') {
+      return hallMan()
+    } else if (sex === 'Woman') {
+      return hallWoman()
+    } else {
+      alert('Sorry try again!')
+      hall()
+    }
+}
+
+function billiardRoom () {
+  var sex = prompt(`At that time, in the Billiard Room, Colonel Mustard, Mr. Green,
+      Ms. Scarlet, Mrs. White, and you were searching for a cigar lighter and
+      then you were killed.
+
+      Do you remember if the killer was a [Man] or a [Woman]`)
+    if (sex === 'Man') {
+      return billiardRoomMan()
+    } else if (sex === 'Woman') {
+      return billiardRoomWoman()
+    } else {
+      alert('Sorry try again!')
+      billiardRoom()
+    }
+}
+
+function conservatoryWoman () {
+  var accused = prompt(`We're almost there! What weapon did the killer use?
+
+  Mrs. Peacock with the [Knife]?
+  Mrs. White with the [Rope]`)
+
+  if (accused === 'Knife' || 'knife') {
```

Comment:
It looks like you're trying to compare the variable `accused` to the strings `'Knife'` and `'knife'`, but this statement doesn't work quite the way you're intending.

Because of operator precedence, it will be evaluated first as `accused === 'Knife'` and then if that fails, the string `'knife'` will be evaluated as a boolean, which will be `true` like all non-empty strings.

In order to test a variable against two strings you'll need to use the `===` operator for each variable/string combination:

```js
accused === 'Knife' || accused === 'knife'
```